metrics.json

```diff
@@ -0,0 +1,44 @@
+{
+  "last": 14400,
+  "sampling": 600,
+  "filter": "ibm_resource_name = 'test123'",
```

Comment:
I assume this is something we'll want to merge in dynamically in the `perf_capture_metrics` method?